packages/xo-web/src/common/modal.js

```diff
@@ -202,7 +213,12 @@ class StrongConfirm extends Component {
           <input
             className='form-control'
             ref={ref => {
-              ref && ref.focus()
+              if (ref !== null) {
+                ref.focus()
+                ref.addEventListener('keydown', this._handleKeyDown)
+                this.componentWillUnmount = () =>
+                  ref.removeEventListener('keydown', this._handleKeyDown)
```

Comment:
After noticing a weird behavior, I read the documentation a bit further and apparently, React checks if it's the same callback on each render and, if it's not, [it will call the old one with `null` and the new one with the DOM node](https://reactjs.org/docs/refs-and-the-dom.html#caveats-with-callback-refs). So we better not inline the ref callback.